arangod/Sharding/ShardingInfo.cpp

```diff
@@ -478,7 +479,7 @@ std::shared_ptr<std::vector<ShardID>> ShardingInfo::shardListAsShardID() const {
   for (auto const& mapElement : *_shardIds) {
     vector->emplace_back(mapElement.first);
   }
-  sortShardNamesNumerically(*vector);
+  std::sort(vector->begin(), vector->end());
```

Comment:
Would be better if we could turn `vector` into a sorted container.

Reply:
Done